web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -63,10 +63,26 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-
     // Set the form title.
     $form['#title'] = $this->t('GFW');
 
+    $form['range'] = [
+      '#type' => 'daterangepicker',
+      '#DateRangePickerOptions' => [
+        'initial_text' => t('Select date range...'),
```

Comment:
You should incude the trait StringTranslationTrait and use $this->t rather than t